service/adapter/classes/order/OrderAdapter.php

```diff
@@ -14,7 +14,7 @@
  *                               #############
  *                               ############
  *
- * Adyen PrestaShop module
+ * Adyen PrestaShop plugin
```

Comment:
We are going to rename all the plugin to module in a later step so that is why you see my new files with the module and not with the plugin :)
In the moment doesn't matter too much